### ISSUE TITLE
Почему не возвращает больше 400 обьектов

### ISSUE THREAD

**Issue by molofeev** [garakh/kladrapi-php#13]
если указать в настройках запроса вот такую строку
$query->ContentName = '.';
то вернет 400 результатов максимум

**Comment by garakh** [comment]
Да, такое ограничение действительно есть. Оно необходимо, чтобы не перегружать сервер. API используется для того, чтоб пользователя было проще найти нужный адрес при вводе, поэтому с точки зрения практических кейсов выдавать более 50 вариантов нет смысла (никто их не будет просматривать), вместо этого пользователь просто введет еще одну букву и сузит выдачу.

**Comment by molofeev** [comment]
я пытался с помощью вашего API получить города что бы заполнять свои кастомные select это удобно в некоторых случаях, хотелось бы контролировать количество выводимых городов

**Comment by garakh** [comment]
Я понимаю, но к сожалению не через наш API такое не сделать. Если вам нужно загрузить все города, то в целом API и не нужен, можно скачать оригинальные базы КЛАДР или ФИАС и выгрузить оттуда все города
